appwrite/website
# Rewrite Deno Quick start documentation

## What does this PR do?

Rewrites the Deno quick-start doc to reflect new Appwrite updates and local function development

## Test Plan

- Run the app and navigate to `/docs/quick-starts/deno`
- Confirm that text and assets are displayed correctly
- Confirm that tutorial steps work as expected
